litex_boards/platforms/digilent_zybo_z7.py

```diff
@@ -113,12 +157,22 @@ class Platform(Xilinx7SeriesPlatform):
     def __init__(self, variant="z7-20", toolchain="vivado"):
         device = {
             "z7-10": "xc7z010-clg400-1",
-            "z7-20": "xc7z020-clg400-1"
+            "z7-20": "xc7z020-clg400-1",
+            "original": "xc7z010-clg400-1"
         }[variant]
+        ps7_config = ps7_config_variants["common"]
+        if variant == "original":
+            _connectors = _connectors_common + _connectors_original
+            _io = _io_common + _io_original
+            ps7_config.update["original"]
+        else:
+            _connectors = _connectors_common + _connectors_z7
+            _io = _io_common + _io_z7
+            ps7_config.update["z7"]
```

Comment:
`ps7_config.update["original"]` is not working. You must use something like:
```
ps7_config.update(ps7_config_variants["original" if variant == "original" else "z7"])
```

instead of using `update` for connectors and io it's possible to call `Xilinx7SeriesPlatform.__init__` with common part followed by:
```
self.add_extension(_io_original if variant == "original" else _io_z7)
self.add_connector(_connector_original if variant == "original" else _connector_z7)
```

And you save few lines of code :)